applications/integration_test.py

```diff
@@ -133,6 +147,7 @@ def test_all(self):
 
     parser.add_argument('-c', '--test_config', help="Path to the json config file")
     parser.add_argument('-b', '--binary_folder', help="Path to the folder that contains the apps binaries")
+    parser.add_argument('-s', '--single', help="Single test to run")
     parser.add_argument('-t', '--test', help="Runs a single test")
```

Comment:
what is the difference between s and t?

Reply:
i hadn't noticed it - cleaned it up in a recent commit i'll push soon